### ISSUE TITLE
1 single page resists

### ISSUE THREAD

**Issue by asheenlevrai** [michitux/dokuwiki-plugin-move#203]
Hi :)

We have been using the move plugin for quite a long time on our wiki and we have always been very happy with it.
Our wiki currently runs 2018-04-22c "Greebo" and the current version of the plugin.

We have recently created a new page on our wiki and at some point we needed to rename it.
However, it doesn't work. When we press the "rename page" button on the right hand tool-set, nothing happens (the usual popup allowing page renaming isn't triggered).

All other pages can still be renamed without any problem.

I don't know how to troubleshoot this.
Can anyone please help?

Thank you very much in advance for your help.
Best,
-a-

**Comment by phy25** [comment]
Could you look at the JavaScript console to see if there's any error message? Meantime you can move pages through admin interface if that's blocking you

**Comment by asheenlevrai** [comment]
I went back to the page to look at what the console outputs but the buttons "delete page" and "rename page" on the right hand side were gone. these buttons are still present and functional on any other page on the wiki.
I'm currently restarting the server...

**Comment by asheenlevrai** [comment]
After restarting the server the "Delete page" button is there again, but not the "rename page" button.
These buttons are present on other pages on our wiki EXCEPT:
 - I noticed that now the "ename page" button is also missing in another page that links to the "resisting page". Other pages that also link to this page have the button present, though.

I tried renaming the "resisting page" via the admin interface but it tells me the page is locked. this is weird since I can normally edit the page if I need to.
